### PR TITLE
Update billing flow

### LINKED ISSUE #214: Update billing flow
## Feature request

Display user info during update billing info.

### PR DESCRIPTION
**Pull request by nicola-palermo**
### All Submissions:

*   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
*   [x] Does your code has proper inline documentation? 

### Changes proposed in this Pull Request:

Displayed plan info on billing details update.

Closes #214  .

### Other information:

*   [x] Have you added an explanation of what your changes do and why you'd like us to include them?
*   [ ] Have you written new tests for your changes, as applicable?
*   [ ] Have you successfully run tests with your changes locally?

### Changelog entry
Displayed plan info on billing details update.

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/stripe/Components/ColumnTitle.js  2
- src/stripe/Components/InfoPriceColumn.js  2
- src/stripe/Components/PlanTable.js  3
         

Clones added
============
- src/stripe/UpdateForm.js  1
- src/stripe/SubscriptionForm.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5906727&bid=19350811)

File: src/stripe/Components/InfoPriceColumn.js [hunk 1]
```diff
@@ -17,22 +26,38 @@ const InfoPriceColumn = (props) => {
...
-				<SubscribeName name={name} />
-				<p className="mobile-price-info is-hidden-on-desktop">
-					<span className="total">{total} &euro; </span>
-					<span className="valxm">{__('per month', 'wubtitle')}</span>
-					<InfoIcon
-						className="info-icon"
-						onClick={() => setIsOpen(!isOpen)}
+				<ColumnTitle name={name} update={update} />
+				{update ? (
+					''
```
Comment (HyprCoder) [inline src/stripe/Components/InfoPriceColumn.js:31]: change the empty string with null

File: src/stripe/Components/PlanTable.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+						<span className="valxm">
+							{__('per month', 'wubtitle')}
+						</span>
+					</td>
+				) : (
+					<td className="val">
+						<span className="cut-vat">
+							{parseFloat(currentPrice) + parseFloat(taxAmount)}
+							&euro;
+							<span className="cut-line">{/* css only */}</span>
```
Comment (HyprCoder) [inline src/stripe/Components/PlanTable.js:36]:
```suggestion
	<span className="cut-line" />
```

File: src/stripe/Components/PlanTable.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+					</td>
+				)}
+			</tr>
+			{renewal ? (
+				<tr className="total">
+					<td>{__('Automatic renewal', 'wubtitle')}</td>
+					<td className="val">{renewal}</td>
+				</tr>
+			) : (
+				''
```
Comment (HyprCoder) [inline src/stripe/Components/PlanTable.js:48]: change the empty string with null

File: src/stripe/UpdateForm.js [hunk 4]
```diff
@@ -28,6 +29,12 @@ function App() {
 	const [error, setError] = useState(null);
 	const [invoiceValues, setInvoiceValues] = useState(null);
 	const [isBack, setIsBack] = useState(false);
+	const [taxable, setTaxable] = useState(true);
+
+	if (invoicePreValues) {
+		setTaxable(invoicePreValues.tax);
+	}
+
```
Comment (HyprCoder) [inline src/stripe/UpdateForm.js:37]: Change this code using the useEffect hook

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/stripe/Components/PlanTable.js  3
- src/stripe/Components/InfoPriceColumn.js  2
- src/stripe/Components/ColumnTitle.js  2
         

Clones added
============
- src/stripe/SubscriptionForm.js  1
- src/stripe/UpdateForm.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5906727&bid=19350811)

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/stripe/Components/InfoPriceColumn.js  2
- src/stripe/Components/ColumnTitle.js  2
- src/stripe/Components/PlanTable.js  3
         

Complexity decreasing per file
==============================
+ src/stripe/UpdateForm.js  -1
         

Clones added
============
- src/stripe/UpdateForm.js  1
- src/stripe/SubscriptionForm.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5906727&bid=19350811)

**Pull request merged**